packages/vitest/src/defaults.ts

```diff
@@ -24,6 +24,7 @@ export const benchmarkConfigDefaults: Required<
   exclude: defaultExclude,
   includeSource: [],
   reporters: ['default'],
+  includeSamples: false,
```

Comment:
Is this not breaking change? Though benchmarking is experimental feature 🤔

Reply:
Right, this is a breaking change. We could start from default true, but I'm hoping that people haven't really needed to have raw samples. For example, since my last PR https://github.com/vitest-dev/vitest/pull/5398. benchmark `outputJson` already omits raw samples.

Reply:
@sheremet-va have we previously introduced breaking changes for experimental features in minor releases, or also on patch releases?

Reply:
All the time